Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,28 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
+  license "MIT"
+  head "https://github.com/kubenetworks/kubevpn.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["OUTPUT_DIR"] = bin
+    ENV["VERSION"] = version
+    ENV["GIT_COMMIT"] = Utils.git_head
+    ENV["BRANCH"] = Utils.git_branch
+    system "make", "kubevpn"
+  end
+
+  test do
+    system "#{bin}/kubevpn", "version"
+  end
```

Comment:
Could you try with a test that covers some of its functionalities? Checks on version or help text almost never fail, even for broken installations. You may check out our [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) for some useful advice.

Reply:
If we want's to test it , we need a k8s cluster... seems hard to test function

Reply:
We don't have to. Testing even for a failure message is better than this because it shows that the software we've built is actually able to do something.